model/host/host.go

```diff
@@ -758,9 +760,10 @@ func FindAllContainers() ([]Host, error) {
 	return hosts, nil
 }
 
-// find all containers belonging to parent
+// GetContainers finds all the containers belonging to this host
+// errors if this host is not a parent
 func (h *Host) GetContainers() ([]Host, error) {
-	if !h.IsParent {
+	if !h.HasContainers {
 		return nil, errors.New("Host is not a parent")
```

Comment:
ah sorry, this should read "host does not host containers". If you can come up with something less confusing, I'd love that, but the terminology is technically correct.